incubator/command-config/src/main/java/io/aklivity/zilla/runtime/command/config/internal/openapi/http/proxy/OpenApiHttpProxyConfigGenerator.java

```diff
@@ -0,0 +1,519 @@
+            }
+        }
+        return result;
+    }
+
+    private NamespaceConfig createNamespace()
+    {
+        return NamespaceConfig.builder()
+            .name("example")
+            .inject(namespace -> injectGuard(namespace))
+            .inject(namespace -> injectTcpServer(namespace, "http_server0", "tls_server0"))
+            .inject(namespace -> injectTlsServer(namespace))
+            .binding()
+                .name("http_server0")
+                .type("http")
+                .kind(SERVER)
+                .options(HttpOptionsConfig::builder)
+                    .access()
+                        .policy(CROSS_ORIGIN)
+                        .build()
+                    .inject(options -> injectHttpServerOptions(options))
+                    .build()
+                .inject(binding -> injectHttpServerRoutes(binding, "http_client0"))
+                .build()
+            .binding()
+                .name("http_client0")
+                .type("http")
+                .kind(CLIENT)
+                .inject(binding -> injectHttpClientRoute(binding, "tcp_client0", "tls_client0"))
+                .build()
+            .inject(namespace -> injectTlsClient(namespace))
+            .binding()
+                .name("tcp_client0")
+                .type("tcp")
+                .kind(CLIENT)
+                .options(TcpOptionsConfig::builder)
+                    .host("") // env
+                    .ports(new int[]{0}) // env
+                    .build()
+                .build()
+            .inject(namespace -> injectVaults(namespace, isTlsEnabled))
+            .build();
+    }
+
+    private NamespaceConfigBuilder<NamespaceConfig> injectGuard(
+        NamespaceConfigBuilder<NamespaceConfig> namespace)
+    {
+        if (isJwtEnabled)
+        {
+            namespace
+                .guard()
+                    .name("jwt0")
+                    .type("jwt")
+                    .options(JwtOptionsConfig::builder)
+                        .issuer("") // env
+                        .audience("") // env
+                        .key()
+                            .alg("").kty("").kid("").use("").n("").e("").crv("").x("").y("") // env
+                            .build()
+                        .build()
+                    .build();
+        }
+        return namespace;
+    }
+
+    private NamespaceConfigBuilder<NamespaceConfig> injectTcpServer(
+        NamespaceConfigBuilder<NamespaceConfig> namespace,
+        String plainExit,
+        String tlsExit)
+    {
+        namespace
+            .binding()
+                .name("tcp_server0")
+                .type("tcp")
+                .kind(SERVER)
+                .options(TcpOptionsConfig::builder)
+                    .host("0.0.0.0")
+                    .ports(allPorts)
+                    .build()
+                .inject(binding -> injectTcpRoutes(binding, plainExit, tlsExit))
+                .build();
+        return namespace;
```

Comment:
Move `return` to first line, since `namespace.binding()....build()` returns `namespace`.

Reply:
fixed